download/model/base.py

```diff
@@ -16,6 +19,14 @@ def data(self):
         """ Just return the data """
         return self._data
     
+    @property
+    def pl(self):
+        ''' Return a Polars dataframe '''
+        if self._pl is None or self._pl.is_empty():
+            self._pl = pl.DataFrame(self._data, schema=self._pl_dtypes)
+
+        return self._pl
+    
     @property
     def df(self):
```

Comment:
this pandas method should go away?  How about just using `df` as before to name the polars dataframe?  The `pl` prefix is too specific and I think in the long run, we're talking about dataframes, whatever implementation is used.